staging/src/k8s.io/kube-aggregator/pkg/controllers/openapiv3/aggregator/downloader.go

```diff
@@ -54,6 +54,33 @@ type SpecETag struct {
 	etag string
 }
 
+func (s *Downloader) DownloadGroups(handler http.Handler) ([]string, error) {
```

Comment:
We're not using the `Download` method below, can we remove it? Also the package is still called `aggregator`, that seems wrong since there is no aggregation anymore? It's still aggregating some things so maybe that's fine.

Reply:
The Download method will be added back once we add in V2->V3 download and conversion. I figured we could maybe keep it there to see the diff for the old and new download function rather than remove the entire function and add in the entire function again. 

i agree that aggregator is not the most accurate name anymore (we still have some instances of "Aggregator" in the openapi v3 controller code). I wanted to reduce the merge conflicts with the cache busting k/k PR but we can make the name changes in this PR if that is desirable.